ssclj/jar/src/com/sixsq/slipstream/ssclj/resources/job.clj

```diff
@@ -56,18 +59,17 @@
 
 (defn add-impl [{:keys [body] :as request}]
   (a/can-modify? {:acl collection-acl} request)
-  (db/add
-    resource-name
-    (-> body
-        u/strip-service-attrs
-        (crud/new-identifier resource-name)
-        (assoc :resourceURI resource-uri)
-        (assoc :state "QUEUED")
-        (ju/status-changed?)
-        u/update-timestamps
-        (crud/add-acl request)
-        crud/validate)
-    {}))
+  (let [{id :id :as new-job} (-> body
+                                 u/strip-service-attrs
+                                 (crud/new-identifier resource-name)
+                                 (assoc :resourceURI resource-uri)
+                                 (assoc :state "QUEUED")
+                                 (ju/status-changed?)
+                                 u/update-timestamps
+                                 (crud/add-acl request)
+                                 crud/validate)]
+    (ju/add-job-to-queue id)
```

Comment:
- What is the desired behavior here on errors?
 - Is the ordering correct between adding the job to zookeeper and adding it to elasticsearch?